README.md

```diff
@@ -534,8 +534,7 @@ with `--username` and `--repository`.
 
 To create the pull request, a personal access token is required for API authentication.
 These can be created at [https://github.com/settings/tokens](https://github.com/settings/tokens).
```

Comment:
How exactly do you do this? When I click on that link and try and "Generate new token" there are loads of options that you can select?!

Reply:
Ah, you mean the different scopes? This is just to limit what can be done with the auth token for security. The simplest way is just to click all of the tick boxes ✅✅✅

I think that only `repo` is required though, and probably not even all of those.